app/app/settings.py

```diff
@@ -102,8 +112,12 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
```

Comment:
I wouldn't worry about changing this at the moment, since everyone running local *should* be using `local_settings.py.dist as .py` and these defaults don't align with the current local dev setup.